mmdet/models/mask_heads/fcn_mask_head.py

```diff
@@ -170,14 +170,22 @@ def get_seg_masks(self, mask_pred, det_bboxes, det_labels, rcnn_test_cfg,
                 mask_pred_ = mask_pred[i, label, :, :]
             else:
                 mask_pred_ = mask_pred[i, 0, :, :]
-            im_mask = np.zeros((img_h, img_w), dtype=np.uint8)
 
             bbox_mask = mmcv.imresize(mask_pred_, (w, h))
             bbox_mask = (bbox_mask > rcnn_test_cfg.mask_thr_binary).astype(
                 np.uint8)
-            im_mask[bbox[1]:bbox[1] + h, bbox[0]:bbox[0] + w] = bbox_mask
-            rle = mask_util.encode(
-                np.array(im_mask[:, :, np.newaxis], order='F'))[0]
-            cls_segms[label - 1].append(rle)
+
+            if rcnn_test_cfg.get("box_rel_masks"):
```

Comment:
1. Single quote is preferred.
2. It is better to set the default value to bool type instead of None.
3. The two field may be renamed to `crop_mask` (False) and `rle_mask_encode` (True)?

Reply:
1. Done
2. Done
3. Yes!)

Reply:
Nice work!